front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -208,6 +221,51 @@ export const usePathfinding = (
     }
   }, [origin?.id, destination?.id, rollingStock]);
 
+  const handleInvalidPathItems = (
+    pathfindingResult: PathfindingResult,
+    pathSteps: (PathStep | null)[],
+    dispatch: AppDispatch,
+    pathfindingDispatch: React.Dispatch<PathfindingAction>,
+    t: TFunction
+  ) => {
```

Comment:
As you are in the component, you can simplify the arguments of your function:
```suggestion
  const handleInvalidPathItems = (
    pathSteps: (PathStep | null)[],
    invalidPathItems: Extract<PathfindingInputError, { error_type: 'invalid_path_items' }>['items']
  ) => {
```

Reply:
done